nncf/onnx/quantization/quantize_model.py

```diff
@@ -75,3 +82,91 @@ def quantize_impl(
     quantized_model = quantization_algorithm.apply(model, graph, dataset=calibration_dataset)
 
     return quantized_model
+
+
+def quantize_with_accuracy_control_impl(
```

Comment:
Waiting for a check of this method on yolov8.
Also, I would like to have a test on that method, probably mocking validation methods and testing on output model graph or output ignored scope....

Reply:
Results are presented below.